end-to-end-tests/cypress/e2e/OSCAL/url_navigation.cy.js

```diff
@@ -0,0 +1,85 @@
+const CATALOG_NAVIGATION =
+  "NIST Special Publication 800-53 Revision 5: Security and Privacy Controls for Federal Information Systems and Organizations";
+const CATALOG_URL_NAV = "/catalog/613fca2d-704a-42e7-8e2b-b206fb92b456";
+
+const MONGODB_NAVIGATION = "MongoDB Component Definition Example";
+const MONGODB_URL_NAV =
+  "/component-definition/a7ba800c-a432-44cd-9075-0862cd66da6b";
+
+const COMPONENT_NAVIGATION = "Test Component Definition";
+const COMPONENT_URL_NAV =
+  "/component-definition/8223d65f-57a9-4689-8f06-2a975ae2ad72";
+
+const PROFILE_NAVIGATION_V4 =
+  "NIST Special Publication 800-53 Revision 4 MODERATE IMPACT BASELINE";
+const PROFILE_V4_URL = "/profile/8b3beca1-fcdc-43e0-aebb-ffc0a080c486";
+
+const PROFILE_NAVIGATION_V5 =
+  "NIST Special Publication 800-53 Revision 5 MODERATE IMPACT BASELINE";
+const PROFILE_V5_URL = "/profile/1019f424-1556-4aa3-9df3-337b97c2c856";
+
+const SSP_NAVIGATION = "Enterprise Logging and Auditing System Security Plan";
+const SSP_URL_NAV =
+  "/system-security-plan/cff8385f-108e-40a5-8f7a-82f3dc0eaba8";
+
+describe("The use URLs to navigate", () => {
+  describe("from SSP Editor to", () => {
```

Comment:
Oopsie, there used to be a before/after each which navigated to the SSP Editor page. I'll remove that describe (line 25) since I no longer think its necessary